openapi/openapi-client/README.md

```diff
@@ -0,0 +1,10 @@
+# Celeborn OpenAPI Client
+
+To update the OpenAPI specification
+- just update the specification under `openapi/openapi-client/src/main/openapi3/` and keep the schema definitions consistent between master and worker.
+- Install JDK11+ by whatever mechanism is appropriate for your system, and set that version to be the default Java version (e.g., by setting env variable JAVA_HOME)
+- run the following:
+  ```sh
+  build/mvn -pl openapi/openapi-client clean package -Popenapi-generate
```

Comment:
Does this mean that currently we only support using `mvn` to generate the `openapi-client` source code? Are there any plans to add `sbt` support?

Reply:
sure, we can support sbt as well, with jdk11+ and same openapi generator plugin version

Reply:
I have setup the sbt build. @cfmcgrady